cos/views/views.py

```diff
@@ -500,3 +504,36 @@ def wait_for_turn(request):
         content_type='json',
         body=json_return
     )
+
+@view_config(route_name='waitForNewPlayers', renderer='json')
+def wait_for_new_players(request):
+    """ This returns a list of players when the player count increases.
+
+            Parameters
+            ----------
+            request: Request 
+                - required JSON parameters: "game_id": String
+
+            Returns
+            -------
+
+            Same object as in get_players_in_game()
+    """
+    json_body = request.json_body
+    if 'game_id' in json_body:
+        game_id = json_body['game_id']
+    else:
+        request.response.status = 400
+        return {'error': "'game_id' is a required parameter for this request"}
+
+    if game_id not in request.registry.games.games:
+        request.response.status = 400
+        return {'error': "Requested Game with id '%s' does not exist." % game_id}
+    game = request.registry.games.games[game_id]
+
+    player_count = len(game.turn_order)
+
+    while (player_count < 4) and (player_count == len(game.turn_order)) and not game.game_started:
+        time.sleep(5)
+
```

Comment:
Should this be returning the ability to start the game once 3 people have joined? Maybe not for the first phase but just trying to think through how the join 'cycle' will work.

Reply:
This doesn’t decide when the game can begin, that’s in “/game/startGame”. All this does is wait for more players to join, and since we can’t add more than 4 players, I figured it should return at 4 since we have the maximum. If /game/startGame is called with 3 players while someone is waiting for this to return, it will automatically cause this to return as well.

Reply:
Awesome. Given the other call can be made before this is done waiting, that sounds like it will solve the problem.